exchanges/poloniex/poloniex_futures_types.go

```diff
@@ -0,0 +1,455 @@
+	MarginMode   string       `json:"mgnMode"`
+	PositionSide string       `json:"posSide"`
+	Symbol       string       `json:"symbol"`
+}
+
+// FuturesOrderbook represents an orderbook data for v3 futures instruments
+type FuturesOrderbook struct {
+	Asks          orderbook.LevelsArrayPriceAmount `json:"asks"`
+	Bids          orderbook.LevelsArrayPriceAmount `json:"bids"`
+	Timestamp     types.Time                       `json:"ts"`
+	LastVersionID int64                            `json:"lid"`
+	ID            types.Number                     `json:"id"`
+	Symbol        string                           `json:"s"`
+	CreationTime  types.Time                       `json:"cT"`
+}
+
+// FuturesCandle represents a kline data for v3 futures instrument
+type FuturesCandle struct {
+	LowestPrice  types.Number
+	HighestPrice types.Number
+	OpeningPrice types.Number
+	ClosingPrice types.Number
+	QuoteAmount  types.Number
+	BaseAmount   types.Number
+	Trades       types.Number
+	StartTime    types.Time
+	EndTime      types.Time
+}
+
+// UnmarshalJSON deserializes JSON data into a kline.Candle instance
+func (v *FuturesCandle) UnmarshalJSON(data []byte) error {
+	target := [9]any{&v.LowestPrice, &v.HighestPrice, &v.OpeningPrice, &v.ClosingPrice, &v.QuoteAmount, &v.BaseAmount, &v.Trades, &v.StartTime, &v.EndTime}
+	return json.Unmarshal(data, &target)
+}
+
+// FuturesExecutionInfo represents a V3 futures instruments execution information
+type FuturesExecutionInfo struct {
+	ID           int64        `json:"id"`
+	Price        types.Number `json:"px"`
+	Quantity     types.Number `json:"qty"`
+	Amount       types.Number `json:"amt"`
+	Side         string       `json:"side"`
+	CreationTime types.Time   `json:"cT"`
+}
+
+// LiquidationPrice represents a liquidiation price detail for an instrument
+type LiquidationPrice struct {
+	Symbol                         string       `json:"symbol"`
+	PositionSide                   string       `json:"posSide"`
+	Side                           string       `json:"side"`
+	Size                           types.Number `json:"sz"`
+	PriceOfCommissionedTransaction types.Number `json:"bkPx"`
+	UpdateTime                     types.Time   `json:"uTime"`
+}
+
+// FuturesTickerDetails represents a v3 futures instrument ticker detail
+type FuturesTickerDetails struct {
+	Symbol       string       `json:"s"`
+	OpeningPrice types.Number `json:"o"`
+	LowPrice     types.Number `json:"l"`
+	HighPrice    types.Number `json:"h"`
+	ClosingPrice types.Number `json:"c"`
+	Quantity     types.Number `json:"qty"`
+	Amount       types.Number `json:"amt"`
+	TradeCount   int64        `json:"tC"`
+	StartTime    types.Time   `json:"sT"`
+	EndTime      types.Time   `json:"cT"`
+	DailyPrice   types.Number `json:"dC"`
+	BestBidPrice types.Number `json:"bPx"`
+	BestBidSize  types.Number `json:"bSz"`
+	BestAskPrice types.Number `json:"aPx"`
+	BestAskSize  types.Number `json:"aSz"`
+	MarkPrice    types.Number `json:"mPx"`
+	Timestamp    types.Time   `json:"ts"`
```

Comment:
Missing field `"dN":"BTC/USDT/PERP"`

Reply:
I couldn't find this `dN` field you are pointing

Reply:
@samuael RE: TestGetFuturesMarket

```
Poloniex attempt 1 request path: https://api.poloniex.com/v3/market/tickers?symbol=BTC_USDT_PERP
Poloniex request type: GET
HTTP status: 200 OK, Code: 200
Poloniex raw response: {"code":200,"msg":"Success","data":[{"s":"BTC_USDT_PERP","o":"114734.52","l":"114100.47","h":"116347.16","c":"114352.69","qty":"28556","amt":"3287689.34126","tC":5617,"sT":1761516480000,"cT":1761602902370,"dN":"BTC/USDT/PERP","dC":"-0.0033","bPx":"114357","bSz":"1","aPx":"114357.16","aSz":"4","mPx":"114352.69"}]}
```